cns/cniconflist/generator_windows.go

```diff
@@ -1,27 +1,131 @@
 package cniconflist
 
 import (
-	"errors"
+	"encoding/json"
+
+	"github.com/Azure/azure-container-networking/cni"
+	"github.com/Azure/azure-container-networking/cni/util"
+	"github.com/Azure/azure-container-networking/network"
+	cniTypes "github.com/containernetworking/cni/pkg/types"
+	"github.com/pkg/errors"
 )
 
 var errNotImplemented = errors.New("cni conflist generator not implemented on Windows")
 
+// V4Overlay N/A for Windows
 func (v *V4OverlayGenerator) Generate() error {
 	return errNotImplemented
 }
 
+// DualStackOverlay N/A for Windows
 func (v *DualStackOverlayGenerator) Generate() error {
 	return errNotImplemented
 }
 
 func (v *OverlayGenerator) Generate() error {
-	return errNotImplemented
+	conflist := cniConflist{
+		CNIVersion:  overlaycniVersion,
+		Name:        overlaycniName,
+		AdapterName: "",
+		Plugins: []any{
+			cni.NetworkConfig{
+				Type:         overlaycniType,
+				Mode:         "bridge",
+				Bridge:       "azure0",
+				Capabilities: map[string]bool{"portMappings": true, "dns": true},
+				IPAM: cni.IPAM{
+					Type: network.AzureCNS,
+					Mode: string(util.V4Overlay),
+				},
+				DNS: cniTypes.DNS{
+					Nameservers: []string{"10.0.0.10", "168.63.129.16"},
+					Search:      []string{"svc.cluster.local"},
+				},
+				AdditionalArgs: []cni.KVPair{
+					{
+						Name: "EndpointPolicy",
+						Value: []byte(`{
+							"Type": "OutBoundNAT",
+                        	"ExceptionList": [
+                            	"10.240.0.0/16",
+                            	"10.0.0.0/8"
+                        	]
+						}`),
+					},
+					{
+						Name: "EndpointPolicy",
+						Value: []byte(`{
+							"Type": "ROUTE",
+							"DestinationPrefix": "10.0.0.0/8",
+                        	"NeedEncap": true
```

Comment:
could you fix the indentation spaces here?

Reply:
@nddq it's a mix of spaces and tabs, you should replace it with all tabs